cstool/cstool_xtensa.inc

```diff
@@ -0,0 +1,62 @@
+#include <stdio.h>
```

Comment:
Please add your copyright. Also, why an `inc` file? It tends to break syntax highlighting and language servers.

Reply:
Because I feel like this print insn detail logic is repeated multiple times (cstool_xxx.c, cstest, test_xxx.c, test_xxx.py), and I'm just a little resistant to copying it four times?

Reply:
This is true. But the test files will be gone very soon. I currently rewrite testing completely. So only the cstool will stay there. You can wait for it, it shouldn't take long.